en/material-system/effect-syntax.md

```diff
@@ -177,8 +190,11 @@ vec4 frag () {
   return CCFragOutput(o);
 }
 ```
-So the code can work in both HDR and LDR pipeline.<br>
+
+The code can work in both HDR and LDR pipeline.
```

Comment:
This is a logical reference to the sentence before the code:

> To encapsulate render pipeline complexities, use .... <code example>, **so that** the code can work in both HDR and LDR pipeline.
 
If possible, you can choose a more proper way to present this.

Reply:
I see what you mean, thank you. this was helpful.